config/dev.exs

```diff
@@ -36,8 +36,8 @@ config :phoenix, :stacktrace_depth, 20
 # Configure your database
 config :skillswheel, Skillswheel.Repo,
   adapter: Ecto.Adapters.Postgres,
-  username: "postgres",
-  password: "postgres",
+  username: "jackcarlisle",
+  password: "",
```

Comment:
yeah that makes sense. It might be nicer if we find a way to configure it like we do with environment variables in node. Something to look into for sure